packages/persist-state/src/lib/storage.ts

```diff
@@ -1,6 +1,6 @@
 import { Observable, of } from 'rxjs';
 
-type MaybeAsync<T> = Promise<T> | Observable<T>;
+export type MaybeAsync<T> = Promise<T> | Observable<T>;
```

Comment:
Can you please change it to Async<T>? it's not maybe :)

Reply:
Done